api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/excelFilterEnabled.md

```diff
@@ -2,7 +2,7 @@
 id: dxDataGrid.Options.export.excelFilterEnabled
 type: Boolean
 default: false
-deprecated: 
+deprecated: Since v20.1, we recommend using [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
```

Comment:
```suggestion
deprecated: Since v20.1, we recommend [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
```